tests/tx_generator/aws_generate_txs.py

```diff
@@ -45,26 +55,47 @@ def set_parser():
     
     """
 
-    # Parse cmd arguments
+    # Parse arguments
     parsed_args = set_parser()
-    print(f"\nreceived args:\n{parsed_args}\n")
+    print(f"\narguments received:\n{parsed_args}\n")
+
+    new_acc_num = parsed_args.new_accounts
     tx_count = parsed_args.tx_count
-    amount = 1
+    layer_duration = parsed_args.layer_duration
+    layer_wait = parsed_args.layer_wait
+    is_concurrent = parsed_args.is_concurrent
+    amount = 100
 
     pod_ip = parsed_args.pod_ip
     pod_lst = [{"pod_ip": pod_ip, "name": "AWS_GATEWAY"}]
 
     my_wallet = WalletAPI(None, pod_lst)
+
     # Get TAP initial values
     tap_nonce = my_wallet.get_nonce_value(conf.acc_pub)
     tap_balance = my_wallet.get_balance_value(conf.acc_pub)
-
     if not tap_nonce or not tap_balance:
         print(f"could not resolve nonce/balance, nonce={tap_nonce}, balance={tap_balance}")
 
     # Create an accountant to follow state
     tap_acc = Accountant.set_tap_acc(balance=tap_balance, nonce=tap_nonce)
     acc = Accountant({conf.acc_pub: tap_acc}, tap_init_amount=tap_balance)
     acc.tx_cost = parsed_args.gas_price
+
     # Create new accounts by sending them coins
-    actions.send_coins_to_new_accounts(my_wallet, tx_count, amount, acc, parsed_args.gas_price)
+    actions.send_coins_to_new_accounts(my_wallet, new_acc_num, amount, acc, parsed_args.gas_price)
+
+    if not tx_count:
+        print("\nbye bye!\n")
+        exit(0)
+
+    tts = layer_wait * layer_duration
+    print(f"sleeping for {tts} to enable new state to be processed")
+    sleep_interval = 60
+    while tts >= 0:
+        time.sleep(sleep_interval)
+        tts -= sleep_interval
+        print(f"{tts} seconds remain")
```

Comment:
don't we have a method for this? (`sleep_print_backwards`)

Reply:
we do but it doesn't show on Jenkins.. if we wait 30 mins I would like to see progress